features/lwipstack/lwip_tools.cpp

```diff
@@ -74,6 +74,23 @@ const ip_addr_t *LWIP::get_ipv4_addr(const struct netif *netif)
     return NULL;
 }
 
+const ip_addr_t *LWIP::get_ipv6_link_local_addr(const struct netif *netif)
```

Comment:
Why this is here, instead of `LWIPStack.cpp` where rest of the LWIP class members are.

Reply:
get_ipv6_link_local_addr is a private function of the class.  Public API's name is get_ipv6_link_local_address